src/commit.ts

```diff
@@ -50,20 +50,38 @@ export interface ConventionalCommitMetadata {
  * Footer class containing key, value pairs
  */
 class Footer {
-  token: string;
-  value: string;
+  private _token!: string;
+  private _value!: string;
 
   constructor(token: string, value: string) {
     this.token = token;
+    this.value = value;
```

Comment:
Shouldnt this be `this._value`?

Reply:
Better not; this'll invoke the `set value(value: string)` function (even though it's transparent atm).